.travis.yml

```diff
@@ -2,7 +2,6 @@ language: ruby
 cache: bundler
 sudo: false
 rvm:
-  - 1.9.3-p551
```

Comment:
why do you remove support for this ruby version?

Reply:
the build was failing for the past several months, 1.9.3-p551 is no longer supported